### ISSUE TITLE
The loss is very high and the acc is very low!

### ISSUE THREAD

**Issue by true-zk** [oalieno/asm2vec-pytorch#16]
I think there must be some problems in ur code... 
Whatever dataset used, the model shows a bad performance... 
In fact, I can only get a 0.3 accuracy and the loss is still about 0.2 even after 128 epochs.
![image](https://user-images.githubusercontent.com/75082700/227266610-3b814df1-82fb-49f0-b509-1d77eb1682f4.png)

or I make something wrong if u can get a high acc.

**Comment by AimiP02** [comment]
I encountered the same problem, whether using `findutils O0-O3` or `coreutils O0-O3`, loss only drops to around 0.22, and accuracy is only about 0.35.

**Comment by roxanecohen** [comment]
Hello, 

I have the same problem, with a loss that stagnates around 0.20. @oalieno, have you observed this behavior ? If not, on what dataset have you trained your model ? Thank you very much by advance
